doc/whats-new.rst

```diff
@@ -51,6 +51,8 @@ Bug fixes
 ~~~~~~~~~
 - Fixed a bug in backend caused by basic installation of Dask (:issue:`4164`, :pull:`4318`)
   `Sam Morley <https://github.com/inakleinbottle>`_.
+- Fixed inconsistencies between docstring and functionality for :py.meth:`DataArray.str.get`
+  and :py.meth:`DataArray.str.wrap` (:issue:`4334`). By `Mathias Hauser <https://github.com/mathause>`_.
```

Comment:
```suggestion
  and :py.meth:`DataArray.str.wrap` (:issue:`4334`). By `Mathias Hauser <https://github.com/mathause>`_.
```

Reply:
Needs to be `:py:meth:` instead of `:py.meth`

Reply:
I think I only ever copy pasted this so I never realized there is a colon.